README.rst

```diff
@@ -29,11 +29,14 @@ ticket here: https://code.djangoproject.com/newticket
 
 To get more help:
 
-* Join the ``#django`` channel on ``irc.libera.chat``. Lots of helpful people
-  hang out there. See https://web.libera.chat if you're new to IRC.
+* Join the `Django Discord <https://discord.gg/xcRH6mN4fa>`_. Lots of helpful
+  people hang out there.
 
-* Join the django-users mailing list, or read the archives, at
-  https://groups.google.com/group/django-users.
+* Post on the `Django forum <https://forum.djangoproject.com/>`_ with a
+  description of your problem or query.
+
+* Visit the `Django community <https://www.djangoproject.com/community/>`_
+  page for details of more ways to get in touch with other Django users.
```

Comment:
We could add a forum, but I don't see a reason to remove the IRC channel.

Reply:
Thanks for your feedback! Changed.